src/go/rpk/pkg/cli/cluster/partitions/move_status.go

```diff
@@ -0,0 +1,216 @@
+				out.ConditionalSectionHeaders(map[string]bool{
+					secMove:      true, // we always print this section
+					secReconcile: all,  // we only print this section if -a is passed
+				})...,
+			)
+			sections.Add(secMove, func() {
+				headers := []string{"Namespace-Topic", "Partition", "Moving-from", "Moving-to", "Completion-%", "Partition-size", "Bytes-moved", "Bytes-remaining"}
+				tw := out.NewTable(headers...)
+				defer tw.Flush()
+				for _, tps := range response {
+					tw.PrintStructFields(f(&tps))
+				}
+			})
+
+			sections.Add(secReconcile, func() {
+				var j int
+				for _, p := range response {
+					fmt.Printf("%s\n", p.Ns+"/"+p.Topic+"/"+strconv.Itoa(p.PartitionID))
+					headers := []string{"Node-id", "Core", "Type", "Retry-number", "Revision", "Status"}
+					tw := out.NewTable(headers...)
+					for _, rs := range p.ReconciliationStatuses {
+						var row []interface{}
+						row = append(row, rs.NodeID)
+						for _, s := range rs.Operations {
+							row = append(row, s.Core, s.Type, s.RetryNumber, s.Revision, s.Status)
+						}
+						tw.Print(row...)
+					}
+					tw.Flush()
+					j++
+					if j < len(response) {
+						fmt.Println()
+					}
+				}
+			})
+		},
+	}
+
+	cmd.Flags().BoolVarP(&all, "print-all", "a", false, "Print internal states about movements for debugging")
+	cmd.Flags().BoolVarP(&human, "human-readable", "H", false, "Print the partition size in a human-readable form")
+	cmd.Flags().StringSliceVarP(&partitions, "partition", "p", nil, "Partitions to print the ongoing movements (repeatable)")
+
+	return cmd
+}
+
+// This function returns true when a partition that movement is
+// ongoing is a requested partition by the --partition option.
+func contains(pReq []string, pRes string) bool {
+	for _, p := range pReq {
+		if p == pRes {
+			return true
+		}
+	}
+	return false
+}
+
+const helpListMovement = `Show ongoing partition movements.
+
+By default this command lists all the ongoing partition movements in the cluster.
+Topics can be specified to print the move status of specific topics. By default,
+this command assumes the "kafka" namespace, but you can use a "namespace/" to
+specify internal namespaces.
+
+    rpk cluster partitions move-status
+    rpk cluster partitions move-status foo bar kafka_internal/tx
+
+The "--partition / -p" flag can be used with topics to additional filter
+requested partitions:
+
+    rpk cluster partitions move-status foo bar --partition 0,1,2
+
+The output contains the following columns, where PARTITION-SIZE is in bytes.
+Using -H, it prints the partition size in a human-readable format
+
+    NAMESPACE-TOPIC
+    PARTITION
+    MOVING-FROM
+    MOVING-TO
+    COMPLETION-%
+    PARTITION-SIZE
+    BYTES-MOVED
+    BYTES-REMAINING
+
+Using "--print-all / -a" the command additionally prints "RECONCILIATION STATUSES"
+which reveals internal states on how the ongoing reconciliations work for debugging
+purposes. That is, reported errors don't necessarily mean real problems.
```

Comment:
```suggestion
Using "--print-all / -a" the command additionally prints "RECONCILIATION STATUSES"
which reveals internal states for the ongoing reconciliations for debugging
purposes. Reported errors do not necessarily mean real problems.
```

What does "real problems" mean here, and what is it that we're actually reporting? The last sentence could stand to be a lot clearer in communicating what the takeaway should be for an administrator reviewing the output of this command.

Reply:
Hey @daisukebe, my PR modified the name of the `move-status` file that you wrote. Do you have any suggestions on how to rephrase this part?

I'm definitely not an expert on how the internal status of a reconciliation/movement works.

Reply:
The `RECONCILIATION STATUSES` section prints very detailed / internal state of Raft reconfiguration (aka partition movement). It typically reports something like this but these are common in general. 

```
NODE-ID  CORE  TYPE    RETRY-NUMBER  REVISION  STATUS
1        1     update  20            664       Generic failure occurred during partition operation execution (cluster::errc:52)
2        1     update  20            664       Current node is not a leader for partition (cluster::errc:17)
3        0     update  20            664       Current node is not a leader for partition (cluster::errc:17)
4        1     update  16            664       Current node is not a leader for partition (cluster::errc:17)
```

This will make a user who reads this believe there's a problem at play, but it's not true. Core team needs to improve the wording, but it's not done yet. This was what I meant "do not necessarily mean real problems".

Even though, it's worth reporting the status with `--a` in a case there's something wrong happening and core team wants to review it.

RE: rephrasing,

Hmm, something like this? Wdyt?

current
```
Reported errors do not necessarily mean real problems.
```

new
```
Use the option with advise from Redpanda support.
```

Reply:
"Advice" but otherwise that's great and clearly communicated when to use this. Thanks!

Reply:
Uses don't need to use it unless Redpanda support asks. How about this?
```
Use the option when Redpanda support requests.
```